.npmignore

```diff
@@ -12,6 +12,7 @@
 .github
 .gitignore
 .jshintrc
+.eslintrc.js
```

Comment:
Place this entry in the file alphabetically and remove the entry for `.eslintrc`